### ISSUE TITLE
Can't see or edit Invision Craft shortcuts

### ISSUE THREAD

**Issue by rodnem** [PeachScript/sketch-plugin-monster#15]
To my mind it's because they are not listed in Plugins menu...

**Comment by PeachScript** [comment]
Yes, currently only commands in the Plugins menu can be supported, perhaps that can be supported in the future, I do not know if it is feasible :D

**Comment by RaphyRaph** [comment]
But toggle craft is in the Plugin menu... is that because they do code injection and hack their way in there?
<img width="860" alt="screenshot 1" src="https://cloud.githubusercontent.com/assets/3810802/23402708/71e74f66-fdb5-11e6-9726-70b54ccd0526.png">
